crates/provider-sdk/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "provider-sdk"
+version = "0.1.0"
+edition = "2021"
+
+[features]
+default = [ ]
+otel = ["opentelemetry", "tracing-opentelemetry", "opentelemetry-otlp"]
+
+[dependencies]
```

Comment:
All of these should be using workspace dependencies and not declare custom versions in this crate

Reply:
Some of them can be, sure. But I don't think optional deps are, so we'll have to keep those